README.md

```diff
@@ -18,24 +18,24 @@ their great cross-browser testing tools.
 
 ---
 
-English | [简体中文](./README-zh_CN.md)
+English | [简体中文](./README-zh_CN.md) | [Hindi](./README-in_HN)
```

Comment:
There is no such file?

Reply:
Hi @jvanbruegge , thanks for pointing this out. I just made another PR that contains (Feature) language support for Hindi, which contains this file.